app/src/main/java/com/sdp/swiftwallet/domain/repository/AuthenticatorFirebaseImpl.java

```diff
@@ -82,6 +90,9 @@ public Result signIn(String email, String password, Runnable success, Runnable f
         return Result.SUCCESS;
     }
 
+    /**
+     * Returns the current user
+     */
     @Override
```

Comment:
Here, I think the comment is not at the correct function but should be for the one below

Reply:
thanks i’ll check